Developers/Newcomers_Guide.rst

```diff
@@ -9,10 +9,12 @@ to complete. The newcomer onboarding process is as follows:
 You will start as a newcomer, which is kind of a trial. If you complete the
 following tasks, you will become a developer at coala:
 
-  - merge a ``difficulty/newcomer`` Pull Request
-  - review at least a ``difficulty/newcomer`` Pull Request
-  - merge a ``difficulty/low`` Pull Request
-  - review at least a``difficulty/low`` or higher Pull Request
+  - solve a ``difficulty/newcomer`` issue
+  - open a pull request for your issue and get it merged
+  - review at least one ``difficulty/newcomer`` Pull Request
+  - solve a ``difficulty/low`` issue
+  - open a pull request for your issue and get it merged
```

Comment:
You mean something like?

There are two parts of the onboarding process:
* tackle a `difficulty/newcomer` issue
* tackle a `difficulty/low` issue

and each of them involves following steps:
  - solve a issue marked with the corresponding label
  - open a pull request for your issue and get it merged
  - review at least one Pull Request with the same difficulty
   - solve a ``difficulty/low`` issue
   - open a pull request for your issue and get it merged
   - review at least one ``difficulty/low`` or higher Pull Request